lib/rmt/http_request.rb

```diff
@@ -9,11 +9,12 @@ def set_defaults
 
```

Comment:
There is one more `try` to get rid of. It's one line above this one:

` Typhoeus::Config.verbose = Settings.try(:http_client).try(:verbose)`

Reply:
removed :)